site/content/en/docs/Reference/fleetautoscaler.md

```diff
@@ -79,6 +79,63 @@ spec:
       seconds: 30
 ```
 
+{{% feature publishVersion="1.37.0" %}}
+Counter-based `FleetAutoscaler` specification below and in the {{< ghlink href="examples/counterfleetautoscaler.yaml" >}}example folder{{< /ghlink >}}:
+
+```yaml
```

Comment:
As a reminder: We'll want indivual copies of this content in `/examples` for each of these FleetAutoscaler examples please (it's just a copy past into a new file - same as https://github.com/googleforgames/agones/blob/main/examples/webhookfleetautoscaler.yaml)

Reply:
We already have two yaml files for counter and list🤔

https://github.com/googleforgames/agones/blob/main/examples/counterfleetautoscaler.yaml
https://github.com/googleforgames/agones/blob/main/examples/listfleetautoscaler.yaml

Reply:
Thank you! I 100% missed those.